includes/Customizer.php

```diff
@@ -204,6 +204,32 @@ public function register_customizer_settings( \WP_Customize_Manager $wp_customiz
 			]
 		);
 
+		$wp_customize->add_setting(
+			self::STORY_OPTION . '[circle_size]',
+			[
+				'default' => $theme_support['default-circle-size'],
+				'type'    => 'option',
+			]
+		);
+
+		$wp_customize->add_control(
+			self::STORY_OPTION . '[circle_size]',
+			[
+				'section'         => self::SECTION_SLUG,
+				'label'           => __( 'Circle Size', 'web-stories' ),
+				'type'            => 'number',
+				'choices'         => $this->get_order_choices( $theme_support['order'] ),
+				'input_attrs'     => [
+					'min'  => 80,
+					'max'  => 200,
+					'step' => 5,
+				],
+				'active_callback' => function() {
```

Comment:
this could be a static function.

Reply:
It accesses `$this`, so it can't be marked as static

Reply:
Your right. Leave it as it is then.